### ISSUE TITLE
Binary included in vault-ssh-helper_0.2.0_linux_amd64.zip still shows version 0.1.6

### ISSUE THREAD

**Issue by aditya4k** [hashicorp/vault-ssh-helper#51]
It seems that the binary included here: https://releases.hashicorp.com/vault-ssh-helper/0.2.0/vault-ssh-helper_0.2.0_linux_amd64.zip still shows the version: 0.1.6

```
abc@jmp200:/tmp$ sudo unzip vault-ssh-helper_0.2.0_linux_amd64.zip 
Archive:  vault-ssh-helper_0.2.0_linux_amd64.zip
replace vault-ssh-helper? [y]es, [n]o, [A]ll, [N]one, [r]ename: Y
  inflating: vault-ssh-helper 
       
abc@jmp200:/tmp$ ./vault-ssh-helper -version
vault-ssh-helper v0.1.6
```

Expected Version should be:
`v0.2.0`

However, I can see the diff between 0.1.6 and newly installed 0.2.0:
```
abc@jmp200:/tmp$ ls -lrt .
-rwxr-xr-x 1 root                root                9483866 Aug 19 19:33 vault-ssh-helper
-rwxr-xr-x 1 root                root                9504203 Aug 19 20:59 vault-ssh-helper0.1.6

abc@jmp200:/tmp$ diff vault-ssh-helper vault-ssh-helper0.1.6 
Binary files vault-ssh-helper and vault-ssh-helper0.1.6 differ
```

**Comment by jsp-hashicorp** [comment]
Is this fixed or not?

**Comment by r4ph4r0ck5** [comment]
No... just have a funny workaround

**Comment by aditya4k** [comment]
@jsp-hashicorp 
I don't think the issue is fixed. We had to locally fix this issue. Issue is still the same.
```
xxxxxx@host:~$ ls -lrt vault-ssh-helper*
-rwxr-xr-x 1 root    root 9483866 Aug 19 19:33 vault-ssh-helperv0.2.0
-rwxr-xr-x 1 abc abc  4860215 Aug 19 19:41 vault-ssh-helper_0.2.0_linux_amd64.zip
-rwxr-xr-x 1 abc abc  9504203 Nov 25 19:56 vault-ssh-helperv0.1.6
```
**version checks**
```
xxxxxx@host:~$ sudo ./vault-ssh-helperv0.2.0 -version
vault-ssh-helper v0.1.6
xxxxxx@host:~$ sudo ./vault-ssh-helperv0.1.6 -version
vault-ssh-helper v0.1.6
```
**Binaries are different though**
```
xxxxxx@host:~$ diff vault-ssh-helperv0.1.6 vault-ssh-helperv0.2.0 
Binary files vault-ssh-helperv0.1.6 and vault-ssh-helperv0.2.0 differ

```

**Comment by kalafut** [comment]
Thanks for reporting this. We'll take a look.

**Comment by calvn** [comment]
Thanks for raising this issue! We just released v0.2.1 to address this.
